workers/unity/Packages/com.improbable.gdk.playerlifecycle/Components/ShouldRequestPlayerTag.cs

```diff
@@ -2,7 +2,9 @@
 
 namespace Improbable.Gdk.PlayerLifecycle
 {
-    public struct ShouldRequestPlayerTag : IComponentData
+    public struct ShouldRequestPlayerTag : ISharedComponentData
```

Comment:
Now no longer a tag, as it has data

Reply:
Why is this a shared component now?

Reply:
IComponentData must be blittable, so I can't have a byte[] field (or a string for that matter)

ran into the same issue as defined here:
https://forum.unity.com/threads/trying-to-store-a-string-in-a-blittable-format.544283/